frontend/src/components/profile/info/badgeHandler.ts

```diff
@@ -186,6 +196,14 @@ function getBadgeMessage(level: number, badge: BadgeType): string {
 
     return msg;
 }
+/**
+ * gets the cohort badge message
+ * @param cohort the cohort
+ * @returns the message
+ */
+function getCohortBadgeMessage(cohort: string): string {
+    return `Congratulations! You have graduated to ${cohort}`;
```

Comment:
You get the cohort badge when you graduate from a cohort, not when you move into that cohort